codalab/lib/bundle_cli.py

```diff
@@ -1120,6 +1120,12 @@ def do_workers_command(self, args):
                 help='Upload to this worksheet (%s).' % WORKSHEET_SPEC_FORMAT,
                 completer=WorksheetsCompleter,
             ),
+            Commands.Argument(
+                '-i',
+                '--ignore',
+                help='Name of the file to read exclusion patterns from when archiving.',
```

Comment:
'when archiving' is an implementation detail, should leave out.  I'd say 'Name of file containing patterns matching files to exclude from upload'.